actioncable/actioncable.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'coffee-rails',     '~> 4.1.0'
   s.add_dependency 'faye-websocket',   '~> 0.10.0'
   s.add_dependency 'websocket-driver', '~> 0.6.1'
-  s.add_dependency 'celluloid',        '~> 0.17.2'
+  s.add_dependency 'concurrent-ruby',  '~> 1.0.0'
```

Comment:
We can :fire: this line since Active Support already depends in concurrent-ruby and will be depending for a long time. This way it would be easier to change the dependency requirement in the framework without having to remember to change in two places.